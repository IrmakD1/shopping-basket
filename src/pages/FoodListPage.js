import React from 'react';
import Food from '../containers/Food'

const FoodListPage = () => {
    return (
      <div>
        <Food />
      </div>
    )
}

export default FoodListPage