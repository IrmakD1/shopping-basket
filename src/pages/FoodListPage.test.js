import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FoodListPage from './BasketPage';

describe('FoodList page', () => {
    let renderer;
    let result;
    beforeEach(() => {
      renderer = new ShallowRenderer();
      renderer.render(<FoodListPage />);
      result = renderer.getRenderOutput();
    });
    test('it should render', () => {
      expect(result).toMatchSnapshot();
    });
});