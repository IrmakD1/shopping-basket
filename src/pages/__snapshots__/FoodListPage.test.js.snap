// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodList page it should render 1`] = `
<div>
  <UNDEFINED />
</div>
`;
